docs/recipes/Recipes.md

```diff
@@ -13,6 +13,7 @@
 * [Running An Existing One App Instance Locally](./Running-Existing-App-Locally.md) 🔨
 * [Running In Production](./Running-In-Production.md) 🔨
 * [Monitoring One App](./Monitoring-One-App.md) 📌
+* [Deploying Modules](./Deploying-Modules.md) 🔨
```

Comment:
You could remove the hammer perhaps when this gets merged. #nit

Reply:
Not yet as it is `Deploying Modules` not `Deploying Modules with Now.sh`. I want to keep this open for contribution on other platforms.